charts/incubator/kometa/values.yaml

```diff
@@ -0,0 +1,69 @@
+image:
+  repository: kometateam/kometa
+  tag: latest
```

Comment:
We require non-latest tags

Reply:
Yep, get the tags here -> https://hub.docker.com/r/kometateam/kometa/tags